docs/how-to-guides/openstack-swift.md

```diff
@@ -0,0 +1,22 @@
+Sometimes it is desirable to cache swift storage objects inside the charm for faster processing and making less requests to the swift server.
+
+In order to use this charm along with Openstack swift storage you'll need to do the following:
```

Comment:
How about something like:
"This guide will demonstrate how to deploy this charm along with Openstack swift storage."